browserslist.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = [
-  'iOS >= 10',
+  'iOS >= 12',
```

Comment:
Why are we defining the iOS version as `iOS >= 12` instead of `last 2 iOS versions`?

Reply:
nvm. this drops minor version released for iOS: https://browserl.ist/?q=last+2+iOS+versions%2C+last+2+and_chr+versions%2C+last+2+android+versions%2C+last+2+chrome+versions%2C+last+2+edge+versions%2C+last+2+firefox+versions%2C+last+2+ie+versions%2C+last+2+safari+versions